Description: Disable color in logs
 After the update to env_logger v0.11, virtiofsd started causing virtualized
 systems to hang at boot time. This was determined to be caused to some component
 of the stack incorrectly parsing logs, probably due to ANSI color escape codes.
 Such a component should probably stop parsing the output of env_logger. In the
 meantime, we disable color in logs.
Author: NoisyCoil <noisycoil@tutanota.com>, Luca Boccassi <bluca@debian.org>
Forwarded: no
Last-Update: 2025-01-12
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -52,7 +52,9 @@
 features = ["derive"]
 
 [dependencies.env_logger]
-version = "0.8.4"
+version = "0.11"
+default-features = false
+features = ["humantime", "regex"]
 
 [dependencies.futures]
 version = "0.3"
